notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>Let's see if we can use the automatically generated Jupyter table of contents instead of trying to maintain one by hand, which may get frustrating with edits to the notebook over time.  It's currently a plugin, but will be integrated into the juplyterlab core in 3.0 (which is coming out momentarily)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
oooh yes! that would make a big difference

Reply:
If you don't have it installed already the plugin is called [jupyterlab-toc](https://github.com/jupyterlab/jupyterlab-toc) (you can install it directly from the plugin manager on the left-hand-side of JupyterLab -- it looks like a puzzle piece)